tools/README.md

```diff
@@ -12,13 +16,13 @@ See tools/docker/contrib/jenkins/configuration/init.groovy.d/003-create-jobs.gro
 
 To add or change the jobs, this is where you want to go.  The name of this file is important, as it needs to get run *BEFORE* the 003-register-github-webhooks.groovy included in the basic install.  Scripts are run alphabetically.  The jobs need to be created before the github webhooks are created.  Our jobs script will read secrets and configmaps created during this setup; described below.
 
-## Prerequisites
+### Prerequisites
 
 You should have your 4 OpenShift projects created for you by ???: dev, test, prod, tools.
```

Comment:
really need to find out ??? or remove this text.

Reply:
Changed wording to assume the structure instead.